src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +24,22 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> myList = new ArrayList<>();
```

Comment:
myList is a bit too generic and gives no context about the collection you are working with. favour names which bring context to the data you're working. for instance it could be `products` or `productList`